docs/tempostack/object_storage.md

```diff
@@ -48,6 +48,37 @@ Tempo Operator supports [AWS S3](https://aws.amazon.com/), [Azure](https://azure
         type: s3
   ```
 
+## Azure
+
+### Requirements
+
+* Create a [bucket](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction) on Azure.
+
+### Installation
+
+* Deploy the Tempo Operator to your cluster.
+
+* Create an Object Storage secret with keys as follows:
+
+    ```console
+    kubectl create secret generic tempostack-dev-azure \
+      --from-literal=container="<AZURE_CONTAINER_NAME>" \
+      --from-literal=account_name="<AZURE_ACCOUNT_NAME>" \
+      --from-literal=account_key="<AZURE_ACCOUNT_KEY>"
+    ```
```

Comment:
```suggestion
```console
kubectl create secret generic tempostack-dev-azure \
  --from-literal=container="<AZURE_CONTAINER_NAME>" \
  --from-literal=account_name="<AZURE_ACCOUNT_NAME>" \
  --from-literal=account_key="<AZURE_ACCOUNT_KEY>"
```
```

Reply:
I'll not apply this change to keep consistency with the rest of the document. The document is rendered properly without those changes.